web/templates/user_group_settings/user_group_settings.hbs

```diff
@@ -26,6 +26,17 @@
                     {{group.description}}
                 </span>
             </div>
+            <div class="group-permissions settings-subsection-parent" id="group_permission_settings">
+                <div class="subsection-header">
+                    <h4 class="user_group_setting_subsection_title">{{t "Group permissions" }}
```

Comment:
The font size for this heading seems smaller than what we have for "Stream permissions".

Reply:
I have made changes. Now the size is consistent with that of "Stream permissions".